frontend/src/admin/users/UserAddContainer.js

```diff
@@ -45,7 +45,16 @@ class UserAddContainer extends Component {
 
   loadData () {
     this.props.getFuelSuppliers();
-    this.props.getRoles();
+
+    if (document.location.pathname.indexOf('/admin/') >= 0) {
```

Comment:
Is it possible to choose whether to filter based on user permissions? This seems... fragile.

Reply:
It does seem a bit fragile doesn't it? 

The backend does technically filter it and show you all the roles you can assign users to. So we can go that route, if ever.

But the main intention of this check is to just show the roles that are available for fuel suppliers or government users. So it's more of a logic restriction than a security check.